src/components/UI/ButtonStyled/RadioButtonstyled.js

```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components';
+
+export const Fieldset = styled.fieldset`
+	display: flex;
+	border: none;
+	height: 3rem;
+	display: flex;
+	align-items: center;
+	margin-bottom: 2rem;
+	margin-left: 10vw;
+	padding-left: 2rem;
+	width: 80vw;
```

Comment:
please don't use vw here, this can quickly lead to unintended effects. Best practice is to let the parent element decide how much this element gets from the available space (for example with a grid and padding / max-width on the parent element).

Reply:
Fixed